core/integration/client_tests.go

```diff
@@ -93,6 +96,34 @@ func WithOutgoingFakeAuth(ctx context.Context, userID string) context.Context {
 	return metadata.NewOutgoingContext(ctx, metadata.New(md))
 }
 
+// SaveTestVectors generates test vectors for interoprability testing.
+func SaveTestVectors(env *Env, resps []testdata.GetEntryResponseVector) error {
+	// Output all key material needed to verify the test vectors.
+	domainFile := "../../core/testdata/domain.json"
+	b, err := json.Marshal(env.Domain)
+	if err != nil {
+		return fmt.Errorf("json.Marshal(): %v", err)
+	}
+	var out bytes.Buffer
+	json.Indent(&out, b, "", "\t")
+	if err := ioutil.WriteFile(domainFile, out.Bytes(), 0666); err != nil {
+		return fmt.Errorf("WriteFile(%v): %v", domainFile, err)
+	}
+	out.Reset()
+
+	// Save list of responses
+	respFile := "../../core/testdata/getentryresponse.json"
```

Comment:
Similarly, couldn't this be "../testdata/getentryresponse.json"?

Reply:
See above